docker-compose.yml

```diff
@@ -30,7 +32,7 @@ services:
     ports:
       - "8042:8042"
     environment:
-      - SE_BACKEND_SERVER=http://pylucene:8043
+      - SE_BACKEND_SERVER=https://172.30.0.1:4269/pylucene
```

Comment:
There should not be a fixed IP address here, because then the compose file will not work for other people.

@GWNLekkah you know more about the Docket setup than I do. Is it correct that the engine connects to Traefik here, instead of the API directly?

Reply:
the same thing is done while fetching the predictions. We should change this as well

Reply:
Haven't figured out networking on WSL tbh, this was the only way we could get it to work. But instead of using the actual IP address, I think it's better to use e.g. maestro.localhost

Reply:
I tried it but It didn't work.